packages/web-styles/src/components/tabs.scss

```diff
@@ -33,13 +34,20 @@
 }
 
 .tabs {
+  --tabs-title-bg-opacity: 1;
```

Comment:
Do we want to declare default variables for components within the :root { } selector (but inside this file)? This would enable the whole page to make use of that variable if tabs are included (might be necessary for the page to use this var somewhere else, in this case I don't see it as very useful but I would like to set a precedent).

Reply:
I don't see the point, why should anyone use this variable outside the tabs-component?
I would rather discuss why this is not a scss-variable.
Otherwise I would always prefer to declare css-variables like this in the component itself and not pollute the :root selector with it.
But maybe your arguments will make sense to me if you explain them.